extensions/python/package.json

```diff
@@ -16,14 +16,15 @@
 		"grammars": [{
 			"language": "python",
 			"scopeName": "source.python",
-			"path": "./syntaxes/Python.tmLanguage"
+			"path": "./syntaxes/Python.tmLanguage.json"
 		},{
 			"scopeName": "source.regexp.python",
-			"path": "./syntaxes/Regular Expressions (Python).tmLanguage"
+			"path": "./syntaxes/Regular Expressions (Python).tmLanguage.json"
 		}]
 	},
 	"scripts": {
 		"compile": "gulp compile-extension:python",
-		"watch": "gulp watch-extension:python"
+		"watch": "gulp watch-extension:python",
+		"update-grammar": "node ../../build/npm/update-grammar.js MagicStack/MagicPython grammars/MagicPython.tmLanguage ./syntaxes/Python.tmLanguage.json && node ../../build/npm/update-grammar.js MagicStack/MagicPython grammars/MagicRegExp.tmLanguage './syntaxes/Regular Expressions (Python).tmLanguage.json'"
```

Comment:
I'd suggest to name the second output file MagicRegExp.tmLanguage.json as to not confuse where it came from.
Also if both grammars are from the same repo, you can use a single call to update-grammar (with 5 parameters: repo, grammarRepoPath1, extPath1, grammarRepoPath2, extPath2)